release-notes-2.0.0.md

```diff
@@ -0,0 +1,12 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### New Features
+- US635012: Rebranded as OpenText.
+
+#### Breaking Changes
+- US635012: As part of the rebrand the swagger configuration file has been renamed from `microfocus-config.js` to `opentext-config.js`.
```

Comment:
Afaik, this is not a breaking change to this module, because it is an internal detail that doesn't impact its consumers.

Reply:
Ok, fair enough. I thought that the renaming of the file could be a breaking change but as you say I have found it doesn't impact how it is consumed.